fishtank/src/cluster.test.ts

```diff
@@ -21,6 +24,34 @@ describe('Cluster', () => {
         labels: { 'fishtank.cluster': 'my-test-cluster' },
       })
     })
+
+    it('launches a detached container with node config', async () => {
+      const backend = new Docker()
+      const cluster = new Cluster({ name: 'my-test-cluster', backend })
+
+      const runDetached = jest.spyOn(backend, 'runDetached').mockReturnValue(Promise.resolve())
+
+      const nodeConfig: Partial<ConfigOptions> = {
+        networkId: 0,
+      }
```

Comment:
Note that there is not assertion that actually checks whether this configuration has been correctly written to a file. Is this coming in a future PR?

Reply:
Since current docker cmd() function is mocked, there is no easy way to verify in unit test. I have verified with docker cli. With the cmd() function input, it pass in the node config and set it properly.

Reply:
The code creates a temporary directory and then writes a file into it. This mechanism is not mocked inside this test, so you can check the contents of the file here.

Reply:
I thought you mean check the file with the docker container. I can do that